hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Server.java

```diff
@@ -3152,15 +3152,17 @@ public void run() {
           IOUtils.cleanupWithLogger(LOG, traceScope);
           if (call != null) {
             updateMetrics(call, startTimeNanos, connDropped);
-            ProcessingDetails.LOG.debug(
```

Comment:
@aajisaka mentioned on other PRs.

> If the log level is not debug, toString() is not actually called.
> http://www.slf4j.org/faq.html#logging_performance

So it is unnecessary that add isDebugEnabled here.

Reply:
Thanks @ferhui for your review. I have updated this patch, please help me review it again. Thanks